Sources/OpenAI/OpenAI.swift

```diff
@@ -35,6 +35,7 @@ final public class OpenAI: OpenAIProtocol {
     }
     
     private let session: URLSessionProtocol
+    private var streamingSessions: [NSObject] = []
```

Comment:
You may want [`NSHashTable`](https://developer.apple.com/documentation/foundation/nshashtable) to give it set semantics and broader memory operations.